guide/concepts/plugins/build.md

```diff
@@ -72,9 +72,9 @@ method called ```newMethod()```. You can call the method like ```someChat.newMet
 Middleware allows you to transform payloads as they travel through the system.
 They are executed in order they are assigned.
 
-The only valid properties of the ```middleware``` object are ```send``` and ```broadcast```.
+The only valid properties of the ```middleware``` object are ```emit``` and ```broadcast```.
```

Comment:
```broadcast``` should be ```on```. This was from an earlier time when I used awful names for these things.

https://github.com/pubnub/chat-engine-markdown/blob/master/src/plugin.js#L53

Reply:
Ok sweet, @ianjennings I've refactored all instances of ```.broadcast()``` --> ```.on()```.